packages/core/src/Utils/Analytics/index.ts

```diff
@@ -44,6 +44,9 @@ export const AnalyticsInitializer = async () => {
                         utm_content: ppc_campaign_cookies?.utm_content,
                         domain: window.location.hostname,
                         url: window.location.href,
+                        network_type: (window.navigator as any).connection?.effectiveType,
+                        network_rtt: (window.navigator as any).connection?.rtt,
+                        network_downlink: (window.navigator as any).connection?.downlink,
                     },
```

Comment:
In `global.d.ts` you can add the type of navigator 

```
type Navigator =  {
    connection?: NetworkInformation;
}


type NetworkInformation = {
    effectiveType?: 'slow-2g' | '2g' | '3g' | '4g';
    rtt?: number;
    downlink?: number;
}


interface Window {
    navigator: Navigator;
}
```

Reply:
can we add '5g' also.or should we restrict it like this or add a string?

Reply:
@aswathy-deriv this is the document for the effectiveType
https://developer.mozilla.org/en-US/docs/Web/API/NetworkInformation/effectiveType
It only accepts those that @prince-deriv provided.

Reply:
okay @ali-hosseini-deriv got it